pkg/virtctl/root.go

```diff
@@ -18,10 +19,20 @@ import (
 	"kubevirt.io/kubevirt/pkg/virtctl/vnc"
 )
 
+var programName string
+
 func NewVirtctlCommand() *cobra.Command {
+	programName = "virtctl"
+	if filepath.Base(os.Args[0]) == "kubectl-virt" {
+		programName = "kubectl virt"
+	}
+
+	cobra.AddTemplateFunc("ProgramName", func() string { return programName })
+	cobra.AddTemplateFunc("prepare", func(s string) string { return strings.Replace(s, "{{ProgramName}}", programName, -1) })
```

Comment:
maybe add comment why this is needed (no templating for examples I guess)

Reply:
Good guessed ;-)